astropy/wcs/tests/test_utils.py

```diff
@@ -425,7 +425,7 @@ class FakeFrame(BaseCoordinateFrame):
     mywcs = celestial_frame_to_wcs(frame, projection='CAR')
     assert tuple(mywcs.wcs.ctype) == ('TLON-CAR', 'TLAT-CAR')
     assert mywcs.wcs.radesys == 'ITRS'
-    assert mywcs.wcs.dateobs == Time('J2000').utc.isot
+    assert mywcs.wcs.dateobs == Time('J2000').utc.fits
```

Comment:
Looking at this again, I realize the change is actually not necessary, but it is probably good to mention that `dateobs` will now be '2000-01-01T11:58:55.816' - it would probably be better if the time system was also stored (as it is in FITS).